common/cpp/verify_ias_report/build_ias_certificates_cpp.sh

```diff
@@ -35,7 +35,7 @@ Cleanup () {
 trap 'echo "**ERROR - line $LINENO**"; Cleanup; exit 1' HUP INT QUIT PIPE TERM ERR
 
 #get certificate from Intel
-wget https://software.intel.com/sites/default/files/managed/7b/de/RK_PUB.zip
+curl https://software.intel.com/sites/default/files/managed/7b/de/RK_PUB.zip -o RK_PUB.zip
```

Comment:
Why is wget replaced by curl? Does this address any issue?

Reply:
I just noticed, wget is failing to download certificates from the url and with this change, its downloading certificate.

Reply:
Yes @manju956 . Though I am not too sure what has caused this to happen suddenly. It could be that the remote server is checking for the User-Agent explicitly and allowing only curl.

Reply:
That is strange. There is also a `wget -U agent-string` option to fake the User-Agent: header line.